src/components/LeftSidebar/NewGroupConversation/SetContacts/SetContacts.vue

```diff
@@ -22,20 +22,26 @@
 <template>
 	<div ref="wrapper" class="set-contacts">
 		<!-- Search -->
-		<NcTextField ref="setContacts"
-			v-observe-visibility="visibilityChanged"
-			:value.sync="searchText"
-			type="text"
-			:label="t('spreed', 'Search participants')"
-			:show-trailing-button="isSearching"
-			:trailing-button-label="cancelSearchLabel"
-			@trailing-button-click="abortSearch"
-			@input="handleInput">
-			<Magnify :size="16" />
-			<template #trailing-button-icon>
-				<Close :size="20" />
-			</template>
-		</NcTextField>
+		<div class="set-contacts__form">
+			<NcTextField ref="setContacts"
+				v-observe-visibility="visibilityChanged"
+				:value.sync="searchText"
+				type="text"
+				class="set-contacts__form-input"
+				:label="t('spreed', 'Search participants')"
+				:show-trailing-button="isSearching"
+				:trailing-button-label="cancelSearchLabel"
+				@trailing-button-click="abortSearch"
+				@input="handleInput">
+				<Magnify :size="16" />
+				<template #trailing-button-icon>
+					<Close :size="20" />
+				</template>
+			</NcTextField>
+			<DialpadPanel container=".set-contacts__form" @dial:type="dialType" />
```

Comment:
You should check if SIP is enabled to render the dial pad ?

Reply:
provided by `canModerateSipDialOut` prop